runtime/heiko/src/lib.rs

```diff
@@ -265,8 +265,8 @@ impl pallet_loans::Config for Runtime {
     type Currency = Currencies;
     type PalletId = LoansPalletId;
     type PriceFeeder = Prices;
-    type ReserveOrigin = EnsureRoot<AccountId>;
-    type UpdateOrigin = EnsureRoot<AccountId>;
+    type ReserveOrigin = EnsureRootOrHalfCouncil;
```

Comment:
I think 2/3 is better

Reply:
@kaichaosun @yz89  What do you suggest here?

Reply:
We can just use `EnsureRootOrHalfCouncil` now because it has been used by several modules.